apt/minimization/minimizer.py

```diff
@@ -180,43 +201,110 @@ def fit(self, X=None, y=None):
             self.cells_ = self.cells
         else:
             self.cells_ = {}
-
+        self.categorical_values = {}
 
         # Going to fit
         # (currently not dealing with option to fit with only X and y and no estimator)
         if self.estimator and X is not None and y is not None:
+
+            if self.is_numpy:
+                if not self.quasi_identifiers:
+                    self.quasi_identifiers = [i for i in range(len(self._features))]
+                x_QI = X[:, self.quasi_identifiers]
+                self.quasi_identifiers_features = [self._features[i] for i in self.quasi_identifiers]
+                X = pd.DataFrame(X, columns=self._features)
+            else:
+                if not self.quasi_identifiers:
+                    self.quasi_identifiers = self._features
+                x_QI = X.loc[:, self.quasi_identifiers]
```

Comment:
What do we assume quasi-identifiers contains in the case of a pandas array? If it's indexes and not feature names (as indicated by the param description) then this line should use iloc instead of loc. If we expect feature names, then the description should be updated.

Reply:
I see it's the same in the anonymizer code. So let's simply update the quasi_identifiers param description both here and in anonymizer to say that in case of pandas it's the column names.